### DISCUSSION TITLE
How to show right side of path instead of left?

### DISCUSSION THREAD

**Discussion by AlejandroSanchez90** [ibhagwan/fzf-lua#1463] [General]
when i do grep, if the path is really long, then i wont see the actual file name. is there a way to default to show last part and ... starting part ?

**Comment by ibhagwan** [comment]
There are a few solutions for that:
- Add `--keep-right` to `fzf_opts`
- Add `multiline=true` or `multiline=2` under `grep`
- While fzf is open toggle line wrap by pressing `alt-/` (default, can be modified)

Note that using multiline / line wrap requires one of the latest versions of fzf (forgot which maybe 0.53?), this is how it looks like:
![IMG_2563](https://github.com/user-attachments/assets/3d42c5b6-fb74-4d0b-8462-df80b562b1cb)

**Reply by AlejandroSanchez90** [reply]
adding  to fzf setup funtcion
```lua
fzf_opts = {
    ['--keep-right'] = true
}
```

or

```shell
:lua require('fzf-lua').files({ fzf_opts = {['--keep-right'] = true} })
```

is not working, anything wrong with my setup?

**Reply by ibhagwan** [reply]
> is not working, anything wrong with my setup?

  It should work, perhaps it’s your fzf version? Add `debug=true` to the options and check if the resulting fzf command contains the keep right flag, also remember that it’s meaningless once you start fuzzy matching, it only works when the query prompt is empty  (man fzf), I recommend using a new version of fzf with a wrap toggle.
